<launch>
  <!-- vim: set ft=xml noet : -->
  <!-- example launch script for ArduPilotMega based FCU's -->

  <arg name="fcu_url" default="udp://:14550@"/>
  <arg name="gcs_url" default="" />
  <arg name="tgt_system" default="1" />
  <arg name="tgt_component" default="1" />
  <arg name="log_output" default="screen" />

  <include file="$(find mavros)/launch/node.launch">
    <arg name="pluginlists_yaml" value="$(find mavros)/launch/apm_pluginlists.yaml" />
    <arg name="config_yaml" value="$(find obc_solo)/launch/apm_config.yaml" />

    <arg name="fcu_url" value="$(arg fcu_url)" />
    <arg name="gcs_url" value="$(arg gcs_url)" />
    <arg name="tgt_system" value="$(arg tgt_system)" />
    <arg name="tgt_component" value="$(arg tgt_component)" />
    <arg name="log_output" value="$(arg log_output)" />
  </include>

  <node pkg="obc_solo" type="camera-driver.py" name="solo_camera_driver" ns="gopro"
      	args="$(find obc_solo)/launch/sololink.sdp" />

  <node pkg="image_proc" type="image_proc" name="image_proc" ns="gopro">
    <remap from="image" to="image_raw"/>
  </node>

  <node pkg="camera_driver" type="people_detector"   name="people_detector">
  </node>

  <node pkg="camera_driver" type="blob_detector.py"   name="blob_detector"   ns="gopro">
	<remap from="image" to="image_rect_color"/>
  </node>


</launch>
